app/templates/docs/action.html

```diff
@@ -2,642 +2,927 @@
+              <ul>
+                <li><code class="docutils literal"><span class="pre">badges</span></code></li>
+                <li><code class="docutils literal"><span class="pre">checkItemStates</span></code></li>
+                <li><code class="docutils literal"><span class="pre">closed</span></code></li>
+                <li><code class="docutils literal"><span class="pre">dateLastActivity</span></code></li>
+                <li><code class="docutils literal"><span class="pre">desc</span></code></li>
+                <li><code class="docutils literal"><span class="pre">descData</span></code></li>
+                <li><code class="docutils literal"><span class="pre">due</span></code></li>
+                <li><code class="docutils literal"><span class="pre">email</span></code></li>
+                <li><code class="docutils literal"><span class="pre">idAttachmentCover</span></code></li>
+                <li><code class="docutils literal"><span class="pre">idBoard</span></code></li>
+                <li><code class="docutils literal"><span class="pre">idChecklists</span></code></li>
+                <li><code class="docutils literal"><span class="pre">idLabels</span></code></li>
+                <li><code class="docutils literal"><span class="pre">idList</span></code></li>
+                <li><code class="docutils literal"><span class="pre">idMembers</span></code></li>
+                <li><code class="docutils literal"><span class="pre">idMembersVoted</span></code></li>
+                <li><code class="docutils literal"><span class="pre">idShort</span></code></li>
+                <li><code class="docutils literal"><span class="pre">labels</span></code></li>
+                <li><code class="docutils literal"><span class="pre">manualCoverAttachment</span></code></li>
+                <li><code class="docutils literal"><span class="pre">name</span></code></li>
+                <li><code class="docutils literal"><span class="pre">pos</span></code></li>
+                <li><code class="docutils literal"><span class="pre">shortLink</span></code></li>
+                <li><code class="docutils literal"><span class="pre">shortUrl</span></code></li>
+                <li><code class="docutils literal"><span class="pre">subscribed</span></code></li>
+                <li><code class="docutils literal"><span class="pre">url</span></code></li>
+              </ul>
+            </li>
+          </ul>
+        </li>
+      </ul>
+    </li>
+  </ul>
+</div>
+<div class="section" id="get-1-actions-idaction-card-field">
+  <h2>GET <span class="target" id="actions-idaction-card-field">/1/actions/[idAction]/card/[field]</span><a class="headerlink" href="#get-1-actions-idaction-card-field" title="Permalink to this headline"></a></h2>
+  <ul class="simple">
+    <li><strong>Required permissions:</strong> read</li>
+    <li><strong>Arguments</strong>
+      <u
```

Comment:
This happened to your last comments too. They were off on the wrong lines. But I know what you're talking about. 😀